libr/core/project.c

```diff
@@ -774,6 +772,14 @@ R_API bool r_core_project_save(RCore *core, const char *prj_name) {
 	char *old_prj_name = NULL;
 	r_return_val_if_fail(prj_name && *prj_name, false);
 	script_path = get_project_script_path (core, prj_name);
+	if (core->bin->is_debugger) {
```

Comment:
i dont think rbin should have this variable at all. imho it must be killed in a further refactoring. use cfg.debug instead.